src/utils/Processing.ts

```diff
@@ -203,19 +213,43 @@ export function loadExternalDependencies(
           `    uri: ${dep.uri ?? 'http://my-fhir-ig.org/ImplementationGuide/123'}\n` +
           '    version: current'
       );
-      continue;
+      return Promise.resolve();
+    } else if (EXT_PKG_TO_FHIR_PKG_MAP[dep.packageId]) {
+      // It is a special "virtual" FHIR extensions package indicating we need to load supplemental
+      // FHIR versions to support "implied extensions".
+      if (dep.version !== fhirVersion) {
+        logger.warn(
+          `Incorrect package version: ${dep.packageId}#${dep.version}. FHIR extensions packages ` +
+            "should use the same version as the implementation guide's fhirVersion. Version " +
+            `${fhirVersion} will be used instead. Update the dependency version in ` +
+            'sushi-config.yaml to eliminate this warning.'
+        );
+      }
+      logger.info(
+        `Loading supplemental version of FHIR to support virtual extensions package: ${dep.packageId}`
```

Comment:
I think the word `virtual` is a bit confusing. Would we be better off explaining a bit more that you are `Loading supplemental version of FHIR to support extensions from other versions of FHIR required by package: ${dep.packageId}`. Or something along those lines.

Reply:
I'm not sure I like my message either to be honest, I just know the word virtual confused me, and I feel like most people using SUSHI won't care (and don't really need to care, assuming all goes well) that the extensions are "virtual".

Reply:
Yeah.  I don't love the phrase "implied extensions" and "virtual extensions package" is also a little problematic. It would be easier if the spec gave us some real names for these ideas, but it doesn't.  But I want to say *something*, otherwise the user will likely be confused as to why we are downloading the core FHIR DSTU2 package (or advanced users might wonder why the requested extensions package isn't showing up in their FHIR cache).

Reply:
I think your suggested message is close.  How about this? `Loading supplemental version of FHIR to support extensions from ${dep.packageId}`

Reply:
Yep, that message looks good!

Reply:
Fixed in ce58754